test/PriceOracle.test.ts

```diff
@@ -0,0 +1,60 @@
+import { expect } from './util/chai';
+import { ethers, deployments, getNamedAccounts, getUnnamedAccounts } from 'hardhat';
+
+// Named accounts used by tests.
+let updater: string;
+let deployer: string;
+let others: string[];
+
+// Before the tests, set named accounts and read deployments.
+before(async function () {
+  const namedAccounts = await getNamedAccounts();
+  deployer = namedAccounts.deployer;
+  updater = namedAccounts.stamper;
+  others = await getUnnamedAccounts();
+});
+
+describe('PriceOracle', function () {
+  describe('when deploying contract', function () {
+    beforeEach(async function () {
+      await deployments.fixture();
+    });
+
+    it('should deploy PriceOracle', async function () {
+      const priceOracle = await ethers.getContract('PriceOracle');
+      expect(priceOracle.address).to.be.a('string');
```

Comment:
Consider: `expect(priceOracle.address).to.be.properAddress`.

Reply:
fixed here, and also for the postageStamp contract